src/main/java/com/synopsys/integration/alert/channel/azure/boards/service/AzureBoardsIssueHandler.java

```diff
@@ -160,10 +186,15 @@ protected String getIssueKey(WorkItemResponseModel issueModel) {
     protected IssueTrackerIssueResponseModel createResponseModel(AlertIssueOrigin alertIssueOrigin, String issueTitle, IssueOperation issueOperation, WorkItemResponseModel issueResponse) {
         Integer workItemId = issueResponse.getId();
         Map<String, ReferenceLinkModel> issueLinks = issueResponse.getLinks();
-        ReferenceLinkModel htmlLink = issueLinks.get("html");
-        String uiLink = Optional.ofNullable(htmlLink)
-                            .map(ReferenceLinkModel::getHref)
-                            .orElseGet(this::getIssueTrackerUrl);
+        // the AzureWorkItemReponst does not contain any links when called via the rest API.
```

Comment:
- Typo in "AzureWorkItemReponst". 
- What does "when called via the rest API" mean? In the response from a POST request? I believe it is there for a GET request.
- We need this link for the callback to Black Duck, so we should add a GET here if necessary.

Reply:
the retrieveExistingIssues call in AzureBoardsIssueHandler makes an API call that returns a WorkItemResponse that contains null for the links.  When the createWorkItemRequest method is called in AzureBoardsIssueHandler the links aren't null.  That's the issue I was seeing.